### DISCUSSION TITLE
Does heyoka support 32-bit systems?

### DISCUSSION THREAD

**Discussion by SaorenPu** [bluescarni/heyoka#424] [Q&A] [answered]
I am currently working on a C++project on the Windows platform that relies on 32-bit systems, but I have learned from official documentation that it does not seem to support 32-bit. Is this the fact?

**Comment by bluescarni** [comment, marked as answer]
Hi @SaorenPu 

There is no reason why heyoka should not work on a 32-bit platform, as it does not contain any 64-bit specific code. Keep in mind though that you would also need to compile all the dependencies in 32-bit mode.

Feel free to give it a try and report the results here - if you encounter issues I would consider them bugs to be fixed in heyoka.
